tools/ci.psm1

```diff
@@ -193,25 +193,25 @@ function Invoke-AppVeyorBuild
     Start-PSBuild -CrossGen -PSModuleRestore -Configuration 'Release' -CI -ReleaseTag $releaseTag
 }
 
-# Implements the AppVeyor 'install' step
+# Implements the CI 'install' step
 function Invoke-AppVeyorInstall
 {
     # Make sure we have all the tags
     Sync-PSTags -AddRemoteIfMissing
     $releaseTag = Get-ReleaseTag
 
-    if(Test-DailyBuild){
-        if ($env:BUILD_REASON -eq 'Schedule') {
-            Write-Host "##vso[build.updatebuildnumber]Daily-$env:BUILD_SOURCEBRANCHNAME-$env:BUILD_SOURCEVERSION-$((get-date).ToString("yyyyMMddhhss"))"
+    if(Test-DailyBuild)
+    {
+        if ($env:BUILD_REASON -eq 'Schedule')
+        {
+            Write-Verbose "##vso[build.updatebuildnumber]Daily-$env:BUILD_SOURCEBRANCHNAME-$env:BUILD_SOURCEVERSION-$((get-date).ToString("yyyyMMddhhss"))"
```

Comment:
```suggestion
            Write-Host "##vso[build.updatebuildnumber]Daily-$env:BUILD_SOURCEBRANCHNAME-$env:BUILD_SOURCEVERSION-$((get-date).ToString("yyyyMMddhhss"))"
```

Reply:
Was going to do this, my git has conflicts :(